### DISCUSSION TITLE
Is webstudio AI going to be open source

### DISCUSSION THREAD

**Discussion by Nibezak** [webstudio-is/webstudio-community#65] [Q&A] [answered]
Hey Websters I have a question , is the webstudio AI going to be opensource , or is it going to be closed

**Comment by kof** [comment]
Hey, check out the repository, it is already open source!

**Reply by Nibezak** [reply]
the AI tool as well ? 🙆🏾‍♂️😃

**Comment by kof** [comment, marked as answer]
Yes

**Reply by Nibezak** [reply]
YOU GUYS ARE THE BESTTTTTTT
You are much needed in the world, You got my full support

**Reply by kof** [reply]
Tell your friends :)

https://twitter.com/getwebstudio/status/1717149389334970482
